index.js

```diff
@@ -76,7 +110,17 @@ function format(answers) {
   // wrap body at 100
   const body = wrap(answers.body, 100)
 
-  return (head + '\n\n' + body)
+  let commitmsg = head + '\n\n' + body
+
+  const issues = answers.issues.match(/#\d+/g)
```

Comment:
Ok so I guess you want to trigger Github automatic issue closing, that's a cool addition 👍

Just a small tweak:
```js
const footer = answers.issues.match(/#\d+/g)
  .map(issue => `Closes ${issue}`)
  .join('\n')

return [head, body, footer].join('\n\n')
```

Reply:
If no matches are found match returns `null` and not an empty array. If the footer is empty we would have 2 additional newlines appended at the end, or do they get removed automatically later?

Reply:
Good catch, let's try this then:
```js
const footer = answers.issues.match(/#\d+/g)
  .map(issue => `Closes ${issue}`)
  .join('\n') || []

return [head, body, footer]
  .join('\n\n')
  .trim()
```